pkg/network/tracer/tracer.go

```diff
@@ -139,12 +134,20 @@ func newTracer(cfg *config.Config, telemetryComponent telemetryComponent.Compone
 	}
 
 	if cfg.ServiceMonitoringEnabled {
-		if !usmconfig.IsUSMSupported() {
-			errStr := fmt.Sprintf("Universal Service Monitoring (USM) requires a Linux kernel version of %s or higher. We detected %s", usmconfig.MinimumKernelVersion, currKernelVersion)
+		if err := usmconfig.CheckUSMSupported(cfg); err != nil {
+			if !errors.Is(err, usmconfig.ErrNotSupported) {
+				return nil, err
+			}
```

Comment:
I think that's a change compared to what we have today

Reply:
Removed